### PR TITLE
some fix & lil changes

### PR DESCRIPTION
**Pull request by Fro99666**
- fix mobile if another language than english / added HTTP server & PHP
version + lang / change memory colsto be as mounts / change load to more
readable stuff / change battery old html to new bar style / added in
core shortcut to memory bar & diask bar / Fix notice on some linux in
OS_Linux.php

### CODE REVIEW

**Comment by jrgp** [comment]
PHP and HTTP versions should be considered secret and confidential (to prevent looking up vulns that are applicable to them).

I'm not willing to add them to linfo.

**Comment by Fro99666** [comment]
i figured out, that ID are auto generated with text in title, so css wasn't fit if language was not english !
this time i think i am done ^_^

anyway i go sleep, if there something to change i will check tomorow !
this version is running on http://info.home2.frogg.fr

**Comment by Fro99666** [comment]
i added an option to disable them:
$settings['show']['webservice'] = false; // Might be dangerous/confidential information; disabled by default.
$settings['show']['phpversion'] = false; // Might be dangerous/confidential information; disabled by default.

**Comment by jrgp** [comment]
Fair enough. You will need to move them to the base OS class though rather than having them directly pull their info in the LinfoOutput class.

File: lib/class.LinfoOutput.php [hunk 1]
```diff
@@ -303,6 +314,24 @@ public function htmlOut() {
 	if (!empty($settings['show']['numLoggedIn']) && array_key_exists('numLoggedIn', $info) && $info['numLoggedIn'])
 		$core[] = array($lang['numLoggedIn'], $info['numLoggedIn']);
 
+	// Shortcut view to memory
```
Comment (jrgp) [inline lib/class.LinfoOutput.php]: Please remove this.

File: lib/class.LinfoOutput.php [hunk 2]
```diff
@@ -269,14 +277,17 @@ public function htmlOut() {
 	if (!empty($settings['cpu_usage']) && isset($info['cpuUsage']) && $info['cpuUsage'] !== false)
 		$core[] = array($lang['cpu_usage'], $info['cpuUsage'].'%');
 
+	// System Load
+	if (!empty($settings['show']['load']))
```
Comment (jrgp) [inline lib/class.LinfoOutput.php]: Please remove this. You can already optionally enable load progress bars per core ($settings['cpu_usage']), and the uptime load values are not indicative of "real" load on unix.

File: lib/class.LinfoOutput.php [hunk 3]
```diff
@@ -243,6 +243,14 @@ public function htmlOut() {
 	if (!empty($settings['show']['hostname']))
 		$core[] = array($lang['hostname'], $info['HostName']);
 	
+	//Web server
```
Comment (jrgp) [inline lib/class.LinfoOutput.php]: Make methods for these in class.OS.php and make getter calls to them in class.Linfo.php like with all the other data given.

File: lib/class.LinfoOutput.php [hunk 4]
```diff
@@ -325,22 +355,25 @@ public function htmlOut() {
...
+					<th>'.$lang['percent_used'].'</th>
 				</tr>
 				<tr>
 					<td>'.$info['RAM']['type'].'</td>
-					<td>'.LinfoCommon::byteConvert($info['RAM']['free']).'</td>
-					<td>'.LinfoCommon::byteConvert($info['RAM']['total'] - $info['RAM']['free']).'</td>
 					<td>'.LinfoCommon::byteConvert($info['RAM']['total']).'</td>
+					<td>'.LinfoCommon::byteConvert($info['RAM']['total'] - $info['RAM']['free']).'</td>
+					<td>'.LinfoCommon::byteConvert($info['RAM']['free']).'</td>
+					<td>'.self::generateBarChart(round(($info['RAM']['total'] - $info['RAM']['free'])*100/$info['RAM']['total'])).'</td>
```
Comment (jrgp) [inline lib/class.LinfoOutput.php]: Please add spaces around the math operators to increase visibility

**Comment by Fro99666** [comment]
ok, done ^_^

**Comment by jrgp** [comment]
F*k'n awesome. Merged.

**Pull request merged**